src/components/v2/V2Project/V2ProjectReconfigureModal/index.tsx

```diff
@@ -272,32 +395,33 @@ export default function V2ProjectReconfigureModal({
         },
         onConfirmed() {
           setReconfigureTxLoading(false)
-          onOk()
+          exit()
         },
       },
     )
+
+    if (!txSuccessful) {
```

Comment:
Fixes https://github.com/jbx-protocol/juice-interface/issues/917

Reply:
Yes very nice, been hit with this a few times